docs/api-constructor.md

```diff
@@ -12,7 +12,7 @@
     -   `options.failOnError` **[Boolean][4]** by default halt processing and raise an error when loading invalid images.
          Set this flag to `false` if you'd rather apply a "best effort" to decode images, even if the data is corrupt or invalid. (optional, default `true`)
     -   `options.density` **[Number][5]** number representing the DPI for vector images. (optional, default `72`)
-    -   `options.page` **[Number][5]** page number to extract for multi-page input (GIF, TIFF) (optional, default `0`)
+    -   `options.page` **[Number][5]** page number to extract for multi-page input (GIF, TIFF, PDF) (optional, default `0`)
```

Comment:
These are autogenerated from the source code by running `npm run docs`. Please can you update https://github.com/lovell/sharp/blob/master/lib/constructor.js#L67 too so we don't lose this change next time the docs are refreshed.

Reply:
Done. Maybe there's no reason to keep the generated code in the repository if it is generated...

Reply:
Thanks for updating. The docs are currently hosted on readthedocs, which requires the markdown to be present in the repo, so maintaining JSDocs and automagically generating markdown is currently the lowest-maintenance approach. There's some discussion in #1451 to migrate away from this approach and/or improve things.